tests/e2e/unbonding.go

```diff
@@ -304,8 +304,10 @@ func (s *CCVTestSuite) TestUnbondingNoConsumer() {
 	providerKeeper := s.providerApp.GetProviderKeeper()
 	providerStakingKeeper := s.providerApp.GetE2eStakingKeeper()
 
-	// remove the consumer chain, which was already registered during setup
-	providerKeeper.DeleteConsumerClientId(s.providerCtx(), s.consumerChain.ChainID)
+	// remove all consumer chains, which were already registered during setup
```

Comment:
The only real change to test behavior is here. In order for `onHold` to be false below, we need to delete all client IDs established with the provider